docs/reference_guides/model_libraries/models_extra/temperature_swing_adsorption/fixed_bed_tsa0d.rst

```diff
@@ -0,0 +1,154 @@
+Fixed Bed Temperature Swing Adsorption (TSA)
+=============================================
+
+The fixed bed TSA unit model consists of a fixed bed four-step temperature swing adsorption (TSA) cycle. 
+The model is implemented as a 0-D equilibrium-based shortcut model without axial variation of states and 
+with a semi-analytical solution of the cyclic steady state. The following assumptions were considered in 
+the model implementation:
+
+* The gas phase behaves ideally.
```

Comment:
I think you need to add an assumption here about the presence of N2 as an inert (as the set up of the material balances assumes this).